src/components/Editor.jsx

```diff
@@ -43,11 +43,7 @@ const makeMarker = () => {
   return marker;
 };
 
-const pushValueToCodeMirror = _.once((value, codeMirrorInst) => {
-  if (!codeMirrorInst.getValue() && value) {
-    codeMirrorInst.setValue(value);
-  }
-});
+const pushValueToCodeMirror = (value, codeMirrorInst) => codeMirrorInst.setValue(value);
```

Comment:
Because the CodeMirror will have a value, but it will be that of the older snippet.